### PR TITLE
[lab-numpy ] Joao Eira

### PR DESCRIPTION
**Pull request by joaopbe**

### CODE REVIEW

**Comment by joaopbe** [comment]
Done!

**Comment by ta-data-lis** [comment]
Lab checked OK.


Regarding your question in challenge 15: "do not understand why f is initialized to the same values as a???" Read the documentation about np.empty:

`Returns: Array of uninitialized (arbitrary) data of the given shape, dtype and order. (...) empty, unlike zeros, does not set the array values to zero, and may therefore be marginally faster. On the other hand, it requires the user to manually set all the values in the array, and should be used with caution.`

Basically, whatever was stored in the memory positions that are allocated to the new array is what you get initially.

Very good job! ✨
